docs/reference/modules/ROOT/pages/query-configuration.adoc

```diff
@@ -0,0 +1,86 @@
+= Query Engine Configuration
+
+There are a number of different defaults that can be overriden within the Crux query engine.
```

Comment:
*overridden

worth getting the `flyspell` package enabled in your adoc-mode :)